packages/mdx-embed/package.json

```diff
@@ -7,8 +7,10 @@
     "prettier": "prettier --config .prettierrc.js --ignore-path .prettierignore --write \"**/*.{json,js,ts,tsx}\"",
     "lint": "eslint . --ext .ts,.tsx",
     "lint:fix": "eslint . --ext .ts,.tsx --fix --quiet",
-    "develop": "webpack --watch",
-    "build": "tsc",
+    "build:types": "tsc --emitDeclarationOnly",
+    "build:js": "babel -d dist src --extensions=\".ts,.tsx\"",
+    "build": "npm run build:types && npm run build:js",
+    "develop": "babel -d dist src --extensions=\".ts,.tsx\" -w",
```

Comment:
@spences10  Not sure if we need a `watch` mode yet but i'll remove it later if i need to.